src/color-slider/README.md

```diff
@@ -126,7 +136,8 @@ If you’re only using the CSS file, you should set these yourself.
 
 | Name | Description |
 |------|-------------|
-| `input` | Fired when the color changes and once during initialization |
+| `input` | Fired when the color changes due to user action |
+| `change` | Fired when the color changes for any reason, and once during initialization |
```

Comment:
Do you mean `colorchange`?

Reply:
At the time I wrote this I didn't didn't, but now yes. Instead, we should also support `change`, but fire it when the slider does. I'm wondering if we should have a `valuechange` event as well.

Alternatively, we could get rid of both of them and just use `propchange` 🤔

Reply:
> Alternatively, we could get rid of both of them and just use `propchange` 🤔

IMHO, this can help if one day we decide that users should also be notified (and able to listen to) about all sorts of events, e.g., `spacechange`, `stopschange`, etc.

Reply:
How about this? Should we fix the `Events` table by changing `change` to `colorchange` and adding the `valuechange` event?

Reply:
Yes. We should also have a change event. Can you check if the change event from the slider propagates by listening to it from the outside? I've been meaning to check this for a while. 
I wonder if we should also list `propchange`. In which case, we could add an `onpropchange` prop too.

Reply:
Btw no need to send PRs for commits that just *add* documentation